src/thor/route_matcher.cc

```diff
@@ -191,7 +191,7 @@ bool expand_from_node(const std::shared_ptr<DynamicCost>* mode_costing,
           elapsed.secs = shape[index].epoch_time() - shape[0].epoch_time();
 
         // Add edge and update correlated index
-        path_infos.emplace_back(mode, elapsed.secs, edge_id, 0, elapsed.cost, false);
+        path_infos.emplace_back(mode, elapsed.secs, edge_id, 0, elapsed.cost, false, -1);
```

Comment:
for all of these you needed to delete `false` on the end and replace with `-1` instead of adding it